crt_portal/static/sass/custom/landing.scss

```diff
@@ -161,19 +161,34 @@ $arrow-height: 24px;
     }
   }
 
+  .crt-landing--logo_section {
+    img {
+      height: 60px;
+      @media screen and (max-width: 480px) {
+        height: 45px;
+      }
+    }
+  }
+
   .crt-landing--logo_subheading {
     @extend .usa-footer__logo-subheading;
-    font-size: $theme-text-font-size-sm;
-    @media screen and (max-width: 400px) {
+    font-size: $theme-text-font-size-xs;
+    line-height: 2.25;
+    margin-top: 0.25rem;
+    @media screen and (max-width: 480px) {
```

Comment:
If we're changing back to `480px`, we can use `mobile-lg` as that's shorthand for this responsive width. Not sure why it was `400px` offhand... I think I wrote `400px` but I can't remember why :sweat_smile: